src/components/TransactionFlow/TxReceipt.tsx

```diff
@@ -191,35 +244,53 @@ export default function TxReceipt({
     txConfig.network.id
   );
 
+  // cannot send from web3 or walletconnect wallets because they overwrite gas and nonce inputs.
+  const isSenderAccountPresent =
+    txConfig &&
+    accounts.find(
+      ({ address, wallet }) =>
+        isSameAddress(address, txConfig.senderAccount?.address) &&
+        ![
```

Comment:
Can we abstract this out to a helper? I think we already have one to detect web3